### PR TITLE
Update Imgur.com.xml

### PR DESCRIPTION
**Pull request by pipboy96**

### CODE REVIEW

File: src/chrome/content/rules/Imgur.com.xml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+	<target host="pb.imgur.com" />
+	<target host="press.imgur.com" />
+	<target host="rt.imgur.com" />
+	<target host="s.imgur.com" />
+	<target host="sen.imgur.com" />
+	<target host="sentry.imgur.com" />
+	<target host="status.imgur.com" />
+	<target host="store.imgur.com" />
+	<target host="zip.imgur.com" />
+	<target host="zrk.imgur.com" />
```
[the remark this answers is not in the document]
Comment (Bisaloo) [reply src/chrome/content/rules/Imgur.com.xml]: Verified with http://zrk.imgur.com/r?e=eyJhdiI6ODA3NzcsImF0IjoyMCwiYnQiOjAsImNtIjo2NjUzNTQsImNoIjoxNjgxMiwiY2siOnt9LCJjciI6NDA3NjY4MCwiZGoiOjAsImlpIjoiOTQ1NzAyM2I5YmJlNGZiNGE1ZGM0MjQzNjRlZWQ5YzYiLCJkbSI6NCwiZmMiOjY2OTE3MDgsImZsIjo2NjQ5MTk2LCJpcCI6IjU0LjE2Ny4zOC4xNDciLCJudyI6OTU2MywicGMiOjAsImVjIjowLCJwciI6NTY5MjgsInJ0IjoxLCJzYSI6IjgiLCJzYiI6ImktMDY1ZWE5NzA2MTY1NDMxODEiLCJzcCI6MTYyOTIsInN0IjowLCJ6biI6MTI3NDg5LCJiZiI6dHJ1ZSwicG4iOiJhemtwcmV2aWV3IiwidXIiOiJodHRwczovL2RvY3MuZ29vZ2xlLmNvbS9mb3Jtcy9kL2UvMUZBSXBRTFNkMVVHd0FfUE5aZ21FWG9pOFk2NExPYnJlX0ZZc2FMUzN0U3hiMHMyWkowMmJlUmcvdmlld2Zvcm0jcmVzcG9uc2VzIiwicmUiOjF9&s=Cqzm1H7Nu1mQzBGWKhYKXK4CrpU but that's a terrible test url. Not sure we want to include it.

File: src/chrome/content/rules/Imgur.com.xml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+	<target host="api.imgur.com" />
+	<target host="apidocs.imgur.com" />
+	<target host="backup.imgur.com" />
+	<target host="blog.imgur.com" />
+	<target host="community.imgur.com" />
+	<target host="help.imgur.com" />
+	<target host="i.imgur.com" />
+	<target host="iob.imgur.com" />
+	<target host="m.imgur.com" />
+	<target host="links.msg.imgur.com" />
```
[the remark this answers is not in the document]
Comment (pipboy96) [reply src/chrome/content/rules/Imgur.com.xml]: No, sorry.

File: src/chrome/content/rules/Imgur.com.xml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+	<target host="pb.imgur.com" />
+	<target host="press.imgur.com" />
+	<target host="rt.imgur.com" />
```
[the remark this answers is not in the document]
Comment (pipboy96) [reply src/chrome/content/rules/Imgur.com.xml]: No.

File: src/chrome/content/rules/Imgur.com.xml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+	<target host="s.imgur.com" />
+	<target host="sen.imgur.com" />
+	<target host="sentry.imgur.com" />
+	<target host="status.imgur.com" />
+	<target host="store.imgur.com" />
+	<target host="zip.imgur.com" />
```
[the remark this answers is not in the document]
Comment (pipboy96) [reply src/chrome/content/rules/Imgur.com.xml]: I can't find any.

File: src/chrome/content/rules/Imgur.com.xml [hunk 5]
```diff
@@ -0,0 +1,43 @@
+<!--
+	Self-signed:
+		- browserevents
+
+	Timeout:
+		- go
+		- stack
+		- i.stack
```
[the remark this answers is not in the document]
Comment (pipboy96) [reply src/chrome/content/rules/Imgur.com.xml]: That's... unusual.

**Comment by Bisaloo** [comment]
I can't find any test urls either so I'm okay with merging this once you add `i.stack.imgur.com` (with the necessary exclusion to pass tests).

**Comment by pipboy96** [comment]
@Bisaloo Thanks! If by some chance you have an Imgur account, please test if this fixes #18687.

**Comment by pipboy96** [comment]
@Bisaloo When I tried `curl https://i.stack.imgur.com/` the request succeeded for some reason. Let's see if tests will pass without the exclusion.

**Comment by Bisaloo** [comment]
Thank you!

> If by some chance you have an Imgur account, please test if this fixes #18687.

Sorry, I don't have an account.

**Pull request merged**